LFS171x/fabric-material/tuna-app/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Emacs backup files
+*~
+*#
+.#*
+# Vim file artifacts
+.*.sw*
+# installed platform-specific binaries
+/bin
+.DS_Store
+.project
+
+first-network/channel-artifacts/*.tx
+first-network/channel-artifacts/*.block
+first-network/crypto-config/*
+
+chaincode-docker-devmode/myc.block
```

Comment:
We don't have these files in the repo

Reply:
I'll remove all first-network and chaincode-docker-devmode lines